opencog/cython/opencog/atomspace_details.pyx

```diff
@@ -411,8 +435,10 @@ cdef api object py_atomspace(cAtomSpace *c_atomspace) with gil:
     cdef AtomSpace atomspace = AtomSpace_factory(c_atomspace)
     return atomspace
 
-cdef api object py_atom(UUID uuid, object atomspace):
```

Comment:
I'm not sure, but I think the above should be ``cdef api object py_atom(Handle h, object atomspace):`` instead of UUID